.github/workflows/ansible-test.yml

```diff
@@ -37,11 +37,16 @@ jobs:
         ansible:
           # It's important that Sanity is tested against all stable-X.Y branches
           # Testing against `devel` may fail as new tests are added.
+          # An alternative to `devel` is the `milestone` branch with
+          # gets synchronized with `devel` every few weeks and therefore
+          # is a more stable target.
         # - stable-2.9 # Only if your collection supports Ansible 2.9
           - stable-2.10
           - stable-2.11
           - stable-2.12
-          - devel
+          - stable-2.13
+        # - devel # Only if you prefer bleeding-edge instability
+          - milestone
```

Comment:
@felixfontein WDYT?

Reply:
`milestone` has some one downside, and the way it works at the moment I would avoid using it in collection CI. The problem is around the time when a new stable branch is created, `milestone` is kept at a state over one month from *before* the stable branch, and isn't updated again until the first development phase of the next ansible-core release begins, which is over a month after the new stable branch is created. (Extrapolating from the 2.14 and 2.13 roadmaps.)

In that time, both the new stable branch and the `milestone` branch have the same 2.x version, but the new stable branch can behave very differently, and more imprtantly have different sanity test behavior, than the `milestone` branch. This did happen for `stable-2.13` vs. `milestone`, where `milestone` required different `tests/sanity/ignore-2.13.txt` entries than `stable-2.13` did during that period. Which meant that collections either had to remove `milestone` for this time, or do not add `stable-2.13` to its CI until `milestone` is again advanced (the second is something they really should not do).

Reply:
How about adding it as a commented out option?

Reply:
@felixfontein FWIW, I've reverted this bit and it should be ready now.

Reply:
Having it commented out sounds good. Making folks aware of `milestone` and its upsides and downsides is good. But we can also do that in another PR, I'm going to merge this one now so `stable-2.13` is finally on the list :)

Reply:
Here you go https://github.com/ansible-collections/collection_template/pull/48